### ISSUE TITLE
Captive portal: Logo is not displayed after updating apk

### ISSUE THREAD

**Issue by RedmineQualitrix** [projectOpenRAP/OpenRAP#23]
Steps to reproduce:
1.Connect to the CDN network.
2.Login as admin.
3.Go to captive portal page.
4.Provide the logo and other details and apk as well.
5.Click on update captive portal.
6.Open the mobile browser and type google.com.
7.Check on the mobile device .
8.Observe there is no logo displayed.

**Comment by chetansk** [comment]
Hi Ayub/Badal,

Can you please share the logo file you used for this testing ? 

Thanks
C

**Comment by RedmineQualitrix** [comment]
This is the file which i used for testing : 

![genie](https://user-images.githubusercontent.com/29939605/33265037-0cfc72f4-d396-11e7-8ac6-4a95bfe19ff7.PNG)

**Comment by chetansk** [comment]
Hi Ayub/Badal,

I tried with the file you have provided, it works. However while UT, I figured out, JPEG files do not work. The logo must be png, did you try non png file ? The logo file you have provided above is png and it works !

Thanks
C

**Comment by RedmineQualitrix** [comment]
@chetansk  
Still not able to add a logo
 
Refer to the video attached below:


[Recording #4.zip](https://github.com/projectOpenRAP/OpenRAP/files/1508644/Recording.4.zip)

**Comment by chetansk** [comment]
Thanks, video shows it very clear. But strange, I am not able to re-create the issue. Let me try again..

Thanks
C

**Comment by RedmineQualitrix** [comment]
The issue is fixed, hence closing the issue.

**Comment by RedmineQualitrix** [comment]
The issue is fixed, hence closing the issue.
